_posts/2018-01-18-From-Manual-to-Automatic.md

```diff
@@ -0,0 +1,111 @@
+Automation scripting times were considerably higher than manual test specification, but get progressively this starting discrepancy has been diluted as the number of test executions cycles increases in both scenarios. For this specific study we've conclude that automatic testing becomes worthwhile after the 36th execution cycle when compared with the same number of manual executions cycles, as shown in the following graphic: 
+
+<br><img src="http://enear.github.io./public/qa-article-images/exec_by_cost_graphic.png"><br>
+
+
+If we were to consider three daily test campaigns cycles, software automation testing would start to compensate after twelve days. 
+
+## How to assess what should be automated 
+
+As previously stated, today’s market contains increasingly complex applications that are constantly improving to offer more features to the end consumer. Between other factors when combined with agile development methodologies it is then possible that these continually mutating products achieve lower *time to market* business metrics, however, both existing and new features need always to be tested to ensure that they meet the expected product quality standards.
+Test automation is then a powerful software development ally since it promotes and maintains quality patterns throughout the product´s lifecycle culminating in swifter results in the development cycles without sacrificing the team with quality-related overheads<sup>[2]</sup>.
+
+Elegible product features that are candidates for test automation are usually those that were previously developed, the one that are considered stable or that have been already manually tested and with known results. On the other hand, unstable, temporary, incomplete features, or those whose automation is so complex, expensive, time-consuming and with high maintenance times are usually left out. This is due to the high initial costs of test automation which are only compensated and supported by successive regression test cycles being a medium for long product life cycles<sup>[3]</sup>.
+
+## Functional automation tools 
+One of the first questions that test automation raises refers to which languages, frameworks, libraries and tools can be used.
+To answer this, let´s describe some of them:
+
++ **RobotFramework** An open source test automation framework developed in Python, based on high-level keywords, which even users with rudimentary programming skills can use. Although this framework has been developed with acceptance tests in mind, the same is able a range of layers taht can go from testing Web GUIs, Databases, APIs and others, depending on which library is included in the framework. It is also possible to create custom keywords and libraries using your own framework or language like Java or Python. Another significant advantage is the quality of produced reports. Besides RobotFramework’s Builtin library, there are other relevant external libraries. The most commonly used are: 
```

Comment:
I don't quite understand this part 
`the same is able a range of layers taht can go from testing Web GUIs, Databases, APIs and others, depending on which library is included in the framework.`
but I think it trying to say something like:
`it provides a range of layers that can go from testing Web GUIs, Databases, APIs and others, depending on which library is included in the framework.`

`Another significant advantage is the quality of produced reports.` -> it should be `reports produced` at the end.